### PR TITLE
Issue7

### PR DESCRIPTION
**Pull request by ai-xiaihai**

### CODE REVIEW

**Comment by ai-xiaihai** [comment]
[React Issue 1: Add Footer](https://trello.com/c/XfVGonMc/188-react-issue-1-add-footer)

**Comment by ai-xiaihai** [comment]
[Issue 7: Delete Image](https://trello.com/c/TJ13s1SH/189-issue-7-delete-image)

File: app/controllers/images_controller.rb [hunk 1]
```diff
@@ -4,9 +4,9 @@ def new
   end
 
   def show
-    image = Image.find(params[:id])
-    @link = image.link
-    @tag_list = image.tag_list
+    @image = Image.find(params[:id])
+    # @link = image.link
```
Comment (eldonchen0501) [inline app/controllers/images_controller.rb]: we should get rid of these lines

Comment (ai-xiaihai) [reply app/controllers/images_controller.rb]: got it

**Pull request merged**